src/main.jsx

```diff
@@ -66,6 +66,7 @@ const setupApp = async persistedState => {
   if (__TARGET__ !== 'mobile') {
     cozy.bar.init({
       appName: data.cozyAppName,
+      cozyClient: client,
```

Comment:
We want to do this only for web app?

Reply:
No, is there somewhere else to inject cozy Client ?

Reply:
Thanks for pointing out this, I've just updated the commit.

Reply:
Yes, here: https://github.com/cozy/cozy-banks/blob/master/src/ducks/mobile/utils.js#L24